crates/primitives/starknet/src/crypto/hash/pedersen.rs

```diff
@@ -65,3 +78,39 @@ impl DefaultHasher for PedersenHasher {
         Self
     }
 }
+
+#[test]
+fn dynamic_string_hashing() {
+    use core::str::FromStr;
+
+    let hasher = PedersenHasher::hasher();
+
+    let message = "Hello, madara!!. It is pedersen hash.".to_string(); // 37 bytes
+    let message = message.as_bytes();
+    let hash_value = hasher.hash_bytes(message);
+
+    assert_eq!(
+        hash_value,
+        Felt252Wrapper(
+            FieldElement::from_str("0x03501abfd0e0606ecab6702213a03294b81837e4d48232df3c39a62b88cc6f73").unwrap()
```

Comment:
just curious, what did we use to calculate this value?

Reply:
I convert the `"Hello, madara!!. It is pedersen hash."` to bytes array and split the bytes with chunks.

Reply:
Oh ok, what I meant is did you use any tool/script which does this for you and the got the value?